QUARKUS-2483.md

```diff
@@ -0,0 +1,80 @@
+# QUARKUS-2483 - Kubernetes service binding support for Reactive SQL clients
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2483
+
+PR: https://github.com/quarkusio/quarkus/pull/25657
+
+In Quarkus 2.10 was added Kubernetes service binding support for 5 Reactive SQL Clients extensions.
+
+Clients with the [Production support](https://access.redhat.com/support/offerings/production/soc/):
+- Reactive PostgreSQL client
+- Reactive MySQL client
+
+Clients with the [Technology Preview features support](https://access.redhat.com/support/offerings/techpreview):
+- Reactive MS SQL client
+
+Client with the [Development support](https://access.redhat.com/support/offerings/developer/):
```

Comment:
Just community support, nothing from RH side

```suggestion
Clients with Quarkus community support:
```

Reply:
Done. I didn't know what `PREVIEW` label means, thank you.